File: Beecrowd/Beginner/1000 - 1099/1074 - Even or Odd/Python/1074_EvenorOdd.py
number = int(input())

for _ in range(number):
    value = int(input())

    if value == 0:
        print("NULL")
    elif value % 2 == 0:
        if value > 0:
            print("EVEN POSITIVE")
        else:
            print("EVEN NEGATIVE")
    else:
        if value > 0:
            print("ODD POSITIVE")
        else:
            print("ODD NEGATIVE")

File: Beecrowd/Beginner/1000 - 1099/1074 - Even or Odd/C++/1074_EvenorOdd.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int number;
    cin >> number;

    while (number--) {
        int value;
        cin >> value;

        if (value == 0) cout << "NULL" << endl;
        else if (value % 2 == 0) {
            if (value > 0) cout << "EVEN POSITIVE" << endl;
            else cout << "EVEN NEGATIVE" << endl;
        } else {
            if (value > 0) cout << "ODD POSITIVE" << endl;
            else cout << "ODD NEGATIVE" << endl;
        }
    }
    
    return 0;
}